scss/civicrm/administer/permissions/_access.scss

```diff
@@ -15,6 +17,16 @@
     .help {
       display: none;
     }
+
+    .status.no-popup {
```

Comment:
this looks some generic styling to me, why are we doing it for a specific page?

Reply:
Status is generally converted to popups like below screenshot
![image](https://user-images.githubusercontent.com/26058635/36727889-e4d4d694-1be4-11e8-8c48-11a4f601e3d2.png)

The page we styled are rare when it comes inside such blocks with no other data and we just do styling for that pages exceptionally.
One of such exceptional variation we did at https://github.com/civicrm/org.civicrm.shoreditch/pull/155